server/lib/tasks/speakers.json

```diff
@@ -0,0 +1,1614 @@
+﻿[
```

Comment:
Ah so this is just the demo db seed data.

Do we have a migration strategy for populating / backing up the prod db without leaking people's PII? Perhaps we should visit privately about that.

Reply:
Ah yes, the prod data migration can be a similar script, but with the prod data done directly on the server, not pushed to github :)